docs/team/member-guide.md

```diff
@@ -36,32 +36,39 @@ a quick list to get you started.
   the team. You can put whatever you'd like here, just try to keep it organized :-)
 
 ## How can I help?
-   
+
 As a member of the team, you are encouraged to continue
 helping in the same ways that you already have. Your contributions to
 documentation, code, etc are always welcome, along with anything in the
 ["how can I contribute"](https://jupyterhub-team-compass.readthedocs.io/en/latest/contributing.html)
-guide in the team-compass.
-   
+guide in the team-compass. If you're on the Binder team, then check out
+[the Binder team responsibilities](https://jupyterhub-team-compass.readthedocs.io/en/latest/binder/governance.html#team-responsibilities)
+page for more information.
+
 Don't forget that, as a member of the team, you're representing the community
 when you interact with people (online and offline). Try to keep a friendly, positive
 attitude, and be welcoming and helpful in bringing others into the community
 and answering their questions.
-   
+
 ### Are there any specific responsibilies?
-   
+
 We don't want team membership to
 be a big burden (many of us have one or more other jobs too!) but there are
 a few things that you should do as a new team member:
-   
+
 1. **"Watch" the [team compass repository](https://github.com/jupyterhub/team-compass)**
    so that you're notified when team conversations are happening.
 2. **Semi-regularly attend team meetings**. You can find a calendar of upcoming
    meetings [on the team meetings page](https://jupyterhub-team-compass.readthedocs.io/en/latest/meetings.html).
 3. **Follow the [Binder team responsibilities](https://jupyterhub-team-compass.readthedocs.io/en/latest/binder/governance.html#team-responsibilities) guidelines**.
    While this is scoped for the Binder team, it is a good rubric to follow for
    JupyterHub as well.
-   
+4. **Commit at least 6 months to team membership**. Training and mentoring new
+   team members is an investment by the existing team, and team
```

Comment:
```suggestion
   team members is an investment by the existing team, hence team
```
WDYT? (this also appears above somewhere so if you accept this we need to change that as well)

Reply:
I updated this in one other place too in the latest commit